src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -203,39 +203,47 @@ static int GetNdkApiLevel(string androidNdkPath, string androidApiLevel, Android
 		}
 
 		bool DoExecute () {
-			Log.LogDebugMessage ("Aot:", AndroidAotMode);
-			Log.LogDebugMessage ("  AndroidApiLevel: {0}", AndroidApiLevel);
-			Log.LogDebugMessage ("  AndroidAotMode: {0}", AndroidAotMode);
-			Log.LogDebugMessage ("  AndroidSequencePointsMode: {0}", AndroidSequencePointsMode);
-			Log.LogDebugMessage ("  AndroidNdkDirectory: {0}", AndroidNdkDirectory);
-			Log.LogDebugMessage ("  AotOutputDirectory: {0}", AotOutputDirectory);
-			Log.LogDebugMessage ("  EnableLLVM: {0}", EnableLLVM);
-			Log.LogDebugMessage ("  IntermediateAssemblyDir: {0}", IntermediateAssemblyDir);
-			Log.LogDebugMessage ("  LinkMode: {0}", LinkMode);
-			Log.LogDebugMessage ("  SdkBinDirectory: {0}", SdkBinDirectory);
-			Log.LogDebugMessage ("  SupportedAbis: {0}", SupportedAbis);
-			Log.LogDebugTaskItems ("  ResolvedAssemblies:", ResolvedAssemblies);
-			Log.LogDebugTaskItems ("  AdditionalNativeLibraryReferences:", AdditionalNativeLibraryReferences);
+			LogDebugMessage ("Aot:", AndroidAotMode);
+			LogDebugMessage ("  AndroidApiLevel: {0}", AndroidApiLevel);
+			LogDebugMessage ("  AndroidAotMode: {0}", AndroidAotMode);
+			LogDebugMessage ("  AndroidSequencePointsMode: {0}", AndroidSequencePointsMode);
+			LogDebugMessage ("  AndroidNdkDirectory: {0}", AndroidNdkDirectory);
+			LogDebugMessage ("  AotOutputDirectory: {0}", AotOutputDirectory);
+			LogDebugMessage ("  EnableLLVM: {0}", EnableLLVM);
+			LogDebugMessage ("  IntermediateAssemblyDir: {0}", IntermediateAssemblyDir);
+			LogDebugMessage ("  LinkMode: {0}", LinkMode);
+			LogDebugMessage ("  SdkBinDirectory: {0}", SdkBinDirectory);
+			LogDebugMessage ("  SupportedAbis: {0}", SupportedAbis);
+			LogDebugTaskItems ("  ResolvedAssemblies:", ResolvedAssemblies);
+			LogDebugTaskItems ("  AdditionalNativeLibraryReferences:", AdditionalNativeLibraryReferences);
 
 			bool hasValidAotMode = GetAndroidAotMode (AndroidAotMode, out AotMode);
 			if (!hasValidAotMode) {
-				Log.LogCodedError ("XA3001", "Invalid AOT mode: {0}", AndroidAotMode);
+				LogCodedError ("XA3001", "Invalid AOT mode: {0}", AndroidAotMode);
 				return false;
 			}
 
 			TryGetSequencePointsMode (AndroidSequencePointsMode, out sequencePointsMode);
 
 			var nativeLibs = new List<string> ();
 
-			if (!RunParallelAotCompiler (nativeLibs))
+			var task = ThreadingTasks.Task.Run<bool> ( () => { return RunParallelAotCompiler (nativeLibs); });
```

Comment:
This should span multiple lines:

    var task = ThreadingTasks.Task.Run<bool>(() => {
            return RunParallelAotCompiler (nativeLibs);
    });

Additionally, won't C# properly infer the return type of `bool` without needing to use `Task.Run<bool>()`?